lib/index.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    rules: {
+        "flow-array-type-style": require("./rules/flow-array-style.js"),
+        "flow-no-one-tuple": require("./rules/flow-no-one-tuple.js"),
+        "imports-requiring-flow": require("./rules/imports-requiring-flow.js"),
+        "react-no-subscriptions-before-mount": require("./rules/react-no-subscriptions-before-mount.js"),
+    },
```

Comment:
It took me a while to figure out that we needed to put all of the rules inside a `rules` property.